docs/operate/customize/urns.md

```diff
@@ -1,12 +1,12 @@
-# DNB URNs registration
+# URN registration
 
 *Available since InvenioRDM v11*
 
 !!! info
 
-    The URN registration feature requires that you have a contract with the [German National Library](https://wiki.dnb.de/display/URNSERVDOK/URN-Service+API). You will need to contact urn-support@dnb.de to get an account for the sandbox and the production REST-API.
+    The URN registration feature requires that you have a contract with the [German National Library](https://wiki.dnb.de/display/URNSERVDOK/URN-Service+API) or another URN provider. You will need to contact urn-support@dnb.de to get an account for the sandbox and the production REST-API.
```

Comment:
This won't work with another URN provider: the module is based on a REST API exclusively offered by the German National Library.